File: WorldFeed/Client/src/app/core/Core/DTO/UploadProcessingError.ts
import {IUploadProcessingError} from "../../Public/Models/Interfaces/DTO/IUploadProcessingError";

export class UploadProcessingError implements IUploadProcessingError {
  // [JsonProperty("code")];
  public code: number;

  // [JsonProperty("name")];
  public name: string;

  // [JsonProperty("message")];
  public message: string;
}

File: Sharebook/Server/src/Services/Upload/Sharebook.Upload.API/DTO/UploadProcessingError.cs
namespace Sharebook.Upload.DTO
{
    using Newtonsoft.Json;

    using Sharebook.Common.Public.Models.Interfaces.DTO;

    public class UploadProcessingError : IUploadProcessingError
    {
        [JsonProperty("code")]
        public int Code { get; set; }


        [JsonProperty("name")]
        public string Name { get; set; }


        [JsonProperty("message")]
        public string Message { get; set; }
    }
}
